lib/s3-sync.js

```diff
@@ -106,9 +109,9 @@ module.exports = class S3Sync extends EventEmitter {
         mime.contentType(path.basename(f.path)) ||
         'application/octet-stream'
 
-      let result = await this.s3.upload({ Key, Body, ContentType }).promise()
+      await this.s3.upload({ Key, Body, ContentType }).promise()
+
       // TODO handle an individual upload failure
-      this.announce({ uploaded: this.status.uploaded + 1 })
```

Comment:
this log constantly shows NaN since this.status.uploaded is initially undefined. It doesn't add any value to the log so I removed it.